app/services/tombstone_migrator.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+class TombstoneMigrator
+  # rubocop:disable Metrics/MethodLength
+  def self.call
+    counter = 0
+    total = query_service.custom_queries.count_all_of_model(model: Tombstone)
+    logger = Logger.new(STDOUT)
+    query_service.find_all_of_model(model: Tombstone).each do |resource|
+      counter += 1
+      logger.info("#{counter} / #{total} resources")
+      deletion_marker = DeletionMarker.new(
+        create_at: resource.created_at,
```

Comment:
should this be `created_at`?

Reply:
This might imply there should be some more test assertions on the relevant test.

Reply:
Good catch. And I'm not going to migrate updated_at, because I think this process should count as an update.